language/move-prover/boogie-backend/src/bytecode_translator.rs

```diff
@@ -529,6 +528,7 @@ impl<'env> ModuleTranslator<'env> {
             Label(_, label) => {
                 self.writer.unindent();
                 emitln!(self.writer, "L{}:", label.as_usize());
+                /*
                 // TODO: revisit whether we can express what is needed here on bytecode level
```

Comment:
It is fine to keep this code around until the disabled functions in verify_vector.move have been sorted out.